Editor/Window/ManagedEC2Page.cs

```diff
@@ -151,12 +152,20 @@ private void UpdateGUI()
             }
             else if (stackStatus.IsStackStatusFailed())
             {
-                _statusIndicator.Set(State.Failed, textProvider.Get(Strings.ManagedEC2DeployStatusFailed));
+                var errorText = textProvider.Get(Strings.ManagedEC2DeployStatusFailed);
+                _statusIndicator.Set(State.Failed, errorText);
+                _deployStatusBox.Show(StatusBox.StatusBoxType.Error,  errorText, null, string.Format(Urls.AwsCloudFormationTemplate, _stateManager.Region, _deploymentSettings.CurrentStackInfo.StackId), Strings.LabelDeploymentCloudFormationConsole);
             }
             else if (stackStatus == StackStatus.DeleteInProgress)
             {
                 _statusIndicator.Set(State.InProgress, textProvider.Get(Strings.ManagedEC2DeployStatusDeleting));
             }
+            else if (stackStatus.IsStackStatusRollback())
+            {
+                var errorText = textProvider.GetError(ErrorCode.StackStatusInvalid);
+                _statusIndicator.Set(State.Failed, errorText);
```

Comment:
The status indicator should not be a sentence. I will confirm with UX on the exact wording to update later, but in the meantime please use "Rolling back" for the in progress rollback statuses and "Rolled back" for the completed rollback statuses.

Reply:
Updated